examples/with-fauna/lib/fauna.js

```diff
@@ -1,49 +1,31 @@
-import { GraphQLClient, gql } from 'graphql-request'
-import { resolveDbDomain } from './constants'
+import { Client, fql } from 'fauna'
```

Comment:
You might consider adding:

```
import 'server-only';
```

And installing the package to ensure this only ever gets used on the server (never in the client).